phone-book/src/main.js

```diff
@@ -0,0 +1,119 @@
+
+
+class PhoneBook {
+  constructor() {
+    this.listHeads = { name: 'Name', cname: 'Last name', email:'Email' };
+    this.listContainerSelector = 'main > .container';
+    this.users = [{	name: 'Иван',
+				cname: 'Петров',
+				email: 'IvanPetrov@ec.ua'},
+				{	name: 'Сергей',
+				cname: 'Сергеев',
+				email: 'SergeiSergeev@ec.ua'},
+					{name: 'Иван',
+					cname: 'Иванов',
+					email: 'IvanIvanov@ec.ua'},
+					{name: 'Александр',
+					cname: 'Александров',
+					email: 'AlexAlex@ec.ua'}
+					,
+				{	name: 'Алекс',
+				cname: 'Смирнов',
+				email: 'AlexSmirnov@ec.ua'},
+				{	name: 'Сергей',
+				cname: 'Волков',
+				email: 'VolkovSergey@ec.ua'},
+				{	name: 'Мария',
+				cname: 'Шарапова',
+				email: 'MariyaSharapova@ec.ua'},
+				{	name: 'Александр',
+				cname: 'Винник',
+				email: 'AlexVinnik@ec.ua'},
+				{	name: 'Дарий',
+				cname: 'Смирнов',
+				email: 'DariySmirnov@ec.ua'},
+				{	name: 'Елена',
+				cname: 'Лещенко',
+				email: 'ElenaLeshenko@ec.ua'},
+				{	name: 'Ольга',
+				cname: 'Новикова',
+				email: 'OlgaNovikova@ec.ua'},
+				{	name: 'Наталья',
+				cname: 'Шемякина',
+				email: 'ShemyakinaN@ec.ua'},
+				{	name: 'Анна',
+				cname: 'Донцова',
+				email: 'AnnaDontsova@ec.ua'},
+				{	name: 'Влад',
+				cname: 'Яма',
+				email: 'VladYama@ec.ua'},
+				{	name: 'Кира',
+				cname: 'Воробьева',
+				email: 'Kira1990@ec.ua'},
+				{	name: 'Виктор',
+				cname: 'Кривенко',
+				email: 'ViktorKriv@ec.ua'}
+				]
+  }
+
+  _makeNode(node,text){
+	if(!text){
```

Comment:
I guess it should looks somehow like this

```js
const nodeElement = document.createElement(node)
if(!text) { 
 return nodeElement
}
/* ... */
```

Reply:
Agree///